.stickler.yml

```diff
@@ -0,0 +1,5 @@
+linters:
+  flake8:
+    fixer: true
+fixers:
+  enable: true
```

Comment:
Tego pliku nie powinieneś był modyfikować. Teraz się zrobił konflikt przez co nie da się zmerdżować Twojej zmiany.
Jeżeli Twoją intencją było żeby mieć zaktualizowaną pociągnięty plik konfiguracyjny, żeby stickler sprawdził błędy w tym PR, to powinieneś był:
1. Zrobić synchronizację z głównym repozytorium zgodnie z instrukcjami podlinkowanymi tutaj: https://github.com/jedzej/tietopythontraining-basic
2. Wyjść z branchem lekcji 3 z master HEAD (tak jak zrobiłeś)
3. Wystawić PR (tak jak zrobiłeś)
4. Ucieszyć się, że wyszło 🥇 

Wtedy byś miał wszystko to co na głównym repo + lekcja 3 na osobnym branchu
Zalecam poprawkę.